Dockerfile

```
@@ -27,6 +27,8 @@ RUN apt-get update \
     file \
     zlib1g-dev \
     liblzma-dev \
+    libyaml-dev \
+    libgdbm-dev \
```

```suggestion
    libgdbm-dev \
    bison \
```

(and then order it) seems to get things to install for me locally